src/rust/src/pkcs12.rs

```diff
@@ -107,6 +107,40 @@ pub(crate) fn symmetric_encrypt(
     Ok(ciphertext)
 }
 
+pub(crate) fn symmetric_decrypt(
```

Comment:
Probably this belongs in a utilities file, since its not actually used in pkcs12.

Reply:
OK, will see where I can put this and let you know.

Reply:
Just checked this conversation was resolved, but we didn't agree on anything. Moved the file to `pkcs7.rs` for now, it makes more sense :)

Reply:
I resolved it because it didn't feel like a blocker, but that works!